src/main.rs

```diff
@@ -12,6 +12,8 @@ mod config;
 struct Mqtt<'a> {
     name: &'a str,
     host: &'a str,
+    user: Option<String>,
```

Comment:
Is it possible to use `str` here, analog to `name` and `host`, or does that lead to problems when using `Option<>`?

Reply:
Done.